<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMarkupsPlaceWidget</class>
 <widget class="qSlicerWidget" name="qSlicerMarkupsPlaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Place markups</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkColorPickerButton" name="ColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select the display color.</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="displayColorName">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="PlaceButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
      <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
      </property>
      <property name="toolTip">
      <string>Place a markup point</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
       <normaloff>:/Icons/MarkupsFiducialMouseModePlace.png</normaloff>:/Icons/MarkupsFiducialMouseModePlace.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="DeleteButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="toolTip">
      <string>Delete last added markup point</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
       <normaloff>:/Icons/MarkupsDelete.png</normaloff>:/Icons/MarkupsDelete.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="MoreButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Click for more options</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc">
       <normaloff>:/Icons/SliceMoreOptions.png</normaloff>:/Icons/SliceMoreOptions.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="ActionPlacePersistentPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Place multiple control points</string>
   </property>
   <property name="toolTip">
    <string>Place multiple control points</string>
   </property>
  </action>
  <action name="ActionDeleteAll">
   <property name="icon">
    <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
     <normaloff>:/Icons/MarkupsDeleteAllRows.png</normaloff>:/Icons/MarkupsDeleteAllRows.png</iconset>
   </property>
   <property name="text">
    <string>Delete all markups</string>
   </property>
   <property name="toolTip">
    <string>Delete all markups in the list</string>
   </property>
  </action>
  <action name="ActionUnsetLast">
   <property name="icon">
    <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
     <normaloff>:/Icons/MarkupsUnset.png</normaloff>:/Icons/MarkupsUnset.png</iconset>
   </property>
    <property name="text">
    <string>Unset last point</string>
   </property>
   <property name="toolTip">
    <string>Unset position of the last point placed without deletion</string>
   </property>
  </action>
  <action name="ActionUnsetAll">
   <property name="icon">
    <iconset resource="../../../Resources/qSlicerMarkupsModule.qrc">
     <normaloff>:/Icons/MarkupsUnsetAll.png</normaloff>:/Icons/MarkupsUnsetAll.png
    </iconset>
   </property>
   <property name="text">
    <string>Unset all points</string>
   </property>
   <property name="toolTip">
    <string>Unset position of all points in the list without deletion</string>
   </property>
  </action>
  <action name="ActionVisibility">
   <property name="icon">
    <iconset resource="../../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc">
     <normaloff>:/Icons/Small/SlicerInvisible.png</normaloff>
     <normalon>:/Icons/Small/SlicerVisible.png</normalon>:/Icons/Small/SlicerInvisible.png</iconset>
   </property>
   <property name="text">
    <string>Visibility</string>
   </property>
   <property name="toolTip">
    <string>Toggle markups visibility</string>
   </property>
  </action>
  <action name="ActionLocked">
   <property name="icon">
    <iconset resource="../../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc">
     <normaloff>:/Icons/Small/SlicerUnlock.png</normaloff>
     <normalon>:/Icons/Small/SlicerLock.png</normalon>:/Icons/Small/SlicerUnlock.png</iconset>
   </property>
   <property name="text">
    <string>Locked</string>
   </property>
   <property name="toolTip">
    <string>Toggle markup point positions lock</string>
   </property>
  </action>
  <action name="ActionFixedNumberOfControlPoints">
   <property name="icon">
    <iconset resource="../../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc">
     <normaloff>:/Icons/Small/SlicerPointNumberUnlock.png</normaloff>
     <normalon>:/Icons/Small/SlicerPointNumberLock.png</normalon>:/Icons/Small/SlicerPointNumberUnlock.png</iconset>
   </property>
   <property name="text">
    <string>Point Number Locked</string>
   </property>
   <property name="toolTip">
    <string>Toggle markup point number lock</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
  <include location="../../../Resources/qSlicerMarkupsModule.qrc"/>
  <include location="../../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc"/>
 </resources>
 <connections/>
</ui>
